src/main/play-doc/operation/ConsolidatedLogging.md

```diff
@@ -20,29 +20,37 @@ The following sub-sections describe each log type and how they are distinguished
 
 Bundle events describe what has happened to a bundle in terms of whether it has loaded, been replicated to a node, scaled up or down, whether resources cannot be found to scale and so forth. The following structured data items determine that a log message represents a bundle event:
 
-* data.mdc.bundleId:`$bundleId`* OR data.mdc.bundleId:`$bundleName`
-* data.mdc.tag:conductr*
+* data.mdc.bundleId:`$bundleId` OR data.mdc.bundleId:`$bundleName`
+* data.mdc.tag:conductr
```

Comment:
Yes, I've tested it withouth the `*`. Not sure why we had it in the first place.